src/main/java/net/starlark/java/annot/ParamType.java

```diff
@@ -36,5 +39,5 @@
    */
   // TODO(adonovan): make this an array---a non-breaking change for most clients---
   // ideally of the same length as the number of type parameters of type().
-  Class<?> generic1() default Object.class;
```

Comment:
A change I would like to make is to replace generic1 with an array of Class types for the list of parameters. The assumption that there is one parameter makes no sense for Dicts, for example.

However, it is a surprisingly involved change, especially in its interactions with Copybara (the other application that uses java.starlark.net).